vsketch/sketch_class.py

```diff
@@ -117,6 +117,14 @@ def finalize(self, vsk: Vsketch) -> None:
         """
         raise NotImplementedError()
 
+    def post_finalize(self, vsk: Vsketch, path: pathlib.Path) -> None:
+        """Called after export, optional in subclasses.
+
+        Intended to allow for including post-processing on
+        saved files.
+        """
+        return
```

Comment:
final nit: `pass` is more idiomatic

Reply:
I actually debated on that.  Guess I landed on the wrong side :stuck_out_tongue: